docs/graphql/core/api-reference/schema-metadata-api/table-view.rst

```diff
@@ -304,6 +357,67 @@ Args syntax
      - :ref:`CustomColumnNames`
      - Customise the column fields
 
+.. _set_table_configuration:
+
+set_table_configuration
+-----------------------
+
+``set_table_configuration`` sets the configuration of a table, through which
+an identifier, custom root fields or custom column names of an already tracked
+table can be set. This will **replace** the already present configuration.
```

Comment:
Can we link to the deprecated configuration here?

Reply:
Sure, I'll do that